fe/fe-core/src/main/java/com/starrocks/transaction/LakeTableTxnStateListener.java

```diff
@@ -112,14 +131,20 @@ public void preWriteCommitLog(TransactionState txnState) {
         Preconditions.checkState(txnState.getTransactionStatus() == TransactionStatus.COMMITTED
                 || txnState.getTransactionStatus() == TransactionStatus.PREPARED);
         TableCommitInfo tableCommitInfo = new TableCommitInfo(table.getId());
+        boolean isFirstPartition = true;
         for (long partitionId : dirtyPartitionSet) {
             Partition partition = table.getPartition(partitionId);
-            long version = -1;
-            if (txnState.getTransactionStatus() == TransactionStatus.COMMITTED) {
-                version = partition.getNextVersion();
```

Comment:
the code is used to fix transactional load in pr https://github.com/StarRocks/starrocks/pull/18158.

Reply:
ok, I will rebase and commit it